script.js

```diff
@@ -0,0 +1,58 @@
+let startBtn = document.querySelector("#startButton");
+startBtn.addEventListener("click", playGame);
+//ash says we can use this as a global object
+let gameState = ["", "", "", "", "", "", "", "", ""];
+function incrementCurrentPlayer(currentPlayer) {
+    currentPlayer++;
+    return currentPlayer;
+}
+function decideTurn(currentPlayer, playerMarker) {
+    if (currentPlayer % 2 == 0) {
+        playerMarker = 'o'
+    } else {
+        playerMarker = 'x'
+    }
+    return playerMarker;
+}
+function addToScreen(tile, playerMarker) {
+    tile.innerHTML = playerMarker;
+}
+function alterGameState(gameState, clicked, playerMarker) {
+    if(gameState[clicked] == "") {
```

Comment:
Suggest using a triple === here instead

Reply:
Its still a ==

Using === just makes sure that JS will never do any weirdness with the datatypes here